src/grammar/ast.ts

```diff
@@ -217,14 +217,14 @@ export type AstStatementAssign = {
 };
 
 export type AstAugmentedAssignOperation =
-    | "+"
-    | "-"
-    | "*"
-    | "/"
-    | "%"
-    | "|"
-    | "&"
-    | "^";
+    | { kind: "augmentedassign_operation"; operation: "+" }
```

Comment:
I'm actually not at all convinced we should add these tags to operators and attributes here and below. For instance, I need to collect attributes for a function, I'd traverse only the function definitions and native function declarations. And if you need some info on augmented assignment operator you 100% need to look at the AST node that actually includes meaningful information like the path and the right-hand side expression. I propose to revert this part of the refact and only introduce things like this if there is ever a use case